deepchem/utils/dft_utils/qccalc/scf_qccalc.py

```diff
@@ -0,0 +1,595 @@
+from __future__ import annotations
```

Comment:
Add a link to the original repo to give them credit

Reply:
done